noir-projects/noir-contracts/contracts/stateful_test_contract/src/main.nr

```diff
@@ -81,6 +81,7 @@ contract StatefulTest {
     }
 
     #[aztec(public)]
+    #[aztec(noinitcheck)]
```

Comment:
Just a clarifying question: even if something is flagged `noinitcheck`, the call will if the instance has not been initialized (and is not in the process of being initialized) right?

Reply:
If the instance has _not_ been initialized and the function is flagged `noinitcheck`, then the call will succeed. So this should be used sparingly, and #4937 removes many of the `noinitcheck`s I added all over the place.